src/draft.ts

```diff
@@ -81,6 +86,11 @@ async function invoke(context : Context, args : string) : Promise<ShellResult> {
     }
 }
 
+async function invokeAsync(context : Context, command : string) : Promise<ShellResult> {
+    let cmd = context.binPath + ' ' + command;
+    return await context.shell.exec(cmd);
```

Comment:
This looks the same as `invoke` except without the `checkPresent` call (which I feel we should keep...?).  It is written a bit differently (`return await exec()` rather than `result = exec(); return result;` but I think the differences turn out to be purely stylistic (except for the `checkPresent` one).